Formula/shtools.rb

```diff
@@ -26,7 +26,11 @@ def install
     pkgshare.install "examples/fortran/", "examples/ExampleDataFiles/"
 
     lib.install "lib/libSHTOOLS.a", "lib/libSHTOOLS-mp.a"
-    include.install "include/fftw3.mod", "include/planetsconstants.mod", "include/shtools.mod", "include/ftypes.mod"
+    include.install "include/fftw3.mod",
+                    "include/planetsconstants.mod",
+                    "include/shtools.mod",
+                    "include/ftypes.mod",
+                    "include/shtools.h"
     share.install "man"
```

Comment:
Why not `make install`?

Reply:
If upstream maintains an install method, that method is preferred.

Reply:
Just to be 100% clear, I should install them into directories like `#{prefix}/lib` and `#{prefix}/include`? The makefile can already do this with `make install PREFIX=#{prefix}` so this would be an easy fix.

Reply:
Yes

Reply:
So `make install PREFIX=#{prefix}` does work. _However_, the test suite doesn't work when installing this way because of some inconsistencies of where the test files are placed. This is not easily fixed in the shtools.rb Formula, but will be fixed upstream for the next release in a couple months.

If possible, I would suggest that we merge this update using the old approach (i.e., `lib.install`, `include.install`, etc), and then in the next release we can replace everything with the simpler make install command. The comment below regarding the problem with Apple silicon will similarly be fixed in the following release.